backend/src/main/java/com/huellapositiva/domain/service/VolunteerProfileService.java

```diff
@@ -117,20 +117,20 @@ private void upsertLocation(UpdateProfileRequestDto updateProfileRequestDto, Jpa
         JpaLocation jpaLocation = JpaLocation.builder()
                 .surrogateKey(surrogateKey)
                 .id(id)
-                .province(updateProfileRequestDto.getProvince())
-                .town(updateProfileRequestDto.getTown())
-                .address(updateProfileRequestDto.getAddress())
-                .island(updateProfileRequestDto.getIsland())
-                .zipCode(updateProfileRequestDto.getZipCode()).build();
+                .province(profileRequestDto.getProvince())
+                .town(profileRequestDto.getTown())
+                .address(profileRequestDto.getAddress())
+                .island(profileRequestDto.getIsland())
+                .zipCode(profileRequestDto.getZipCode()).build();
         jpaVolunteer.setLocation(jpaLocation);
     }
 
     /**
      * This method update information in profile table
-     *  @param updateProfileRequestDto New user credential information to update
-     * @param jpaVolunteer      Email of user logged
+     *  @param profileRequestDto New user credential information to update
+     *  @param jpaVolunteer      JPA representation of the volunteer
      */
-    private void upsertProfile(UpdateProfileRequestDto updateProfileRequestDto, JpaVolunteer jpaVolunteer) {
+    private void upsertVolunteerProfile(UpdateVolunteerProfileRequestDto profileRequestDto, JpaVolunteer jpaVolunteer) {
```

Comment:
https://github.com/ayudadigital/huelladigital-backend/pull/94#discussion_r606176690

This method can be simplified with the same principle.

Reply:
Done.